app/js/lib/foxbox/network.js

```diff
@@ -56,7 +56,7 @@ export default class Network {
     return Promise.resolve();
   }
 
-  get origin() {
+  get url() {
```

Comment:
Why renaming `origin` by `url`? AFAIK a URL is something like `example.com`, while an origin is `https://example.com:443`.